### PR TITLE
refactor(ertp): jessie-check where easy

### PR DESCRIPTION
**Pull request by erights**
Adds @jessie-check directives to ERTP src/**/*.js files when doing so has zero problems. 

See 
https://github.com/Agoric/agoric-sdk/pull/3876
https://github.com/Agoric/agoric-sdk/pull/5497
https://github.com/Agoric/agoric-sdk/pull/7486
https://github.com/Agoric/agoric-sdk/pull/7481

### CODE REVIEW

File: packages/ERTP/src/paymentLedger.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// @jessie-check
```
Comment (dckc) [inline packages/ERTP/src/paymentLedger.js:1]: :clap: 

Nice to see that this key file was already jessie-ready.

**Review by dckc — COMMENTED** [review]
This looks good except...

> Staged on #7481

Maybe split out the ERTP-specific part of #7481?

**Comment by erights** [comment]
It turns out this PR and #7481 were disjoint, so unstaging.

File: packages/ERTP/src/paymentLedger.js (hunk 1 above)
Comment (erights) [reply packages/ERTP/src/paymentLedger.js:1]: > Maybe split out the ERTP-specific part of https://github.com/Agoric/agoric-sdk/pull/7481?

It turns out that #7481 and this PR were disjoint, so I unstaged. Resolving.

**Review by dckc — APPROVED** [review]
good stuff!

**Pull request merged**